sdk/core/src/main/java/com/klaviyo/coresdk/Klaviyo.kt

```diff
@@ -5,19 +5,43 @@ import com.klaviyo.coresdk.networking.requests.IdentifyRequest
 import com.klaviyo.coresdk.networking.requests.KlaviyoRequest
 import com.klaviyo.coresdk.networking.requests.TrackRequest
 
+/**
+ * Our main API for the Klaviyo SDK.
+ * Where we pass in analytics requests to be sent to the Klaviyo backend
+ */
 object Klaviyo {
+    /**
+     * Creates a track request and sends it off for proceessing.
+     * Track requests track the triggering of a specified [KlaviyoEvent] in Klaviyo
+     *
+     * @param customerProperties A map of customer property information.
+     * Defines the customer that triggered this event
+     * @param properties A map of event property information.
+     * Additional properties associated to the event that are not for identifying the customer
+     */
     fun track(event: KlaviyoEvent, customerProperties: Map<String, String>, properties: Map<String, String>? = null) {
         val request = TrackRequest(event.name, customerProperties.toMutableMap(), properties)
         request.generateUnixTimestamp()
         processRequest(request)
     }
 
+    /**
+     * Creates an identify request and sends it off for processing.
```

Comment:
Might be useful to clarify where it sends it off to (Klaviyo's asynchronous APIs).